test/libethereum/BlockchainTestsFiller/bcGasPricerTestFiller.json

```diff
@@ -30,7 +33,7 @@
                 "storage": {}
             },
             "aaaf5374fce5edbc8e2a8697c15331677e6ebf0b" : {
-                "balance" : "10000000000",
```

Comment:
When you change something different from the `expect` section, please explain why.